content/en/tracing/troubleshooting/tracer_startup_logs.md

```diff
@@ -30,11 +30,18 @@ The Java tracer does not output Diagnostics logs.  For this check, run the trace
 
 #### File Location
 
-Log files are located in `%PROGRAMDATA%\Datadog .NET Tracer\logs`.
```

Comment:
Only other thought here is we should maybe link back to the configuration table in either dotnet-core or dotnet-framework. Wdyt @jaycdave88 ?